courses/machine_learning/intro_to_tensorflow/4_keras_functional_api.ipynb

```diff
@@ -0,0 +1,440 @@
+{
```

Comment:
<p>I'd remove the positional argument "key=" (or "keys=") in the feature columns (it's always required in position 1) to make the code less verbose, e.g., </p><p> </p><p>fc.numeric_column(key="pickup_longitude")  --&gt; fc.numeric_column("pickup_longitude")</p><p> </p><p>fc.crossed_column(keys=[fc_bucketized_dlat, fc_bucketized_dlon], hash_bucket_size=NBUCKETS * NBUCKETS)</p><p> </p><p>--&gt; fc.crossed_column([fc_bucketized_dlat, fc_bucketized_dlon], hash_bucket_size=NBUCKETS * NBUCKETS)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/576/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>